root/opt/nethvoice-report/api/templates/cdr_year.sql

```diff
@@ -4,7 +4,34 @@ CREATE TABLE IF NOT EXISTS `cdr_{{ YearMap .Year }}`
 	`cost` DOUBLE DEFAULT NULL, UNIQUE KEY uniq (calldate,uniqueid,dstchannel,duration),
 	`dispositions` LONGTEXT DEFAULT ''
 )
-SELECT *,
+SELECT `calldate`,
+       `clid`,
+       `src`,
+       `dst`,
+       `dcontext`,
+       `channel`,
+       `dstchannel`,
+       `lastapp`,
+       `lastdata`,
+       MAX(`duration`) as duration,
+       IF (MIN(`disposition`) = 'ANSWERED', MAX(billsec), MIN(billsec)) as billsec,
+       `disposition`,
+       `amaflags`,
+       `accountcode`,
+       `uniqueid`,
+       `userfield`,
+       `did`,
+       `recordingfile`,
+       `cnum`,
+       `cnam`,
+       `outbound_cnum`,
+       `outbound_cnam`,
+       `dst_cnam`,
+       `linkedid`,
+       `peeraccount`,
+       `sequence`,
+       `ccompany`,
+       `dst_ccompany`,
```

Comment:
Could be the code improved as below @edospadoni ?
```suggestion
SELECT *,
       MAX(`duration`) as duration,
       IF (MIN(`disposition`) = 'ANSWERED', MAX(billsec), MIN(billsec)) as billsec,
```

Reply:
Some external queries can read the fields in a specific order. To avoid errors in fields ordering we can preserve this patch with fields explicit

Reply:
we decided to maintain the expanded version to be sure that column order isn't changed and avoid possible regressions